pkg/local-gadget-manager/local-gadget-manager.go

```diff
@@ -348,6 +348,7 @@ func NewManager(runtimes []*containerutils.RuntimeConfig) (*LocalGadgetManager,
 
 	err = l.ContainerCollection.Initialize(
 		containercollection.WithPubSub(containerEventFuncs...),
+		containercollection.WithOCIConfigEnrichment(),
```

Comment:
I think it should also be added to `pkg/gadgettracermanager/gadgettracermanager.go`, otherwise this fix won't be available to Inspektor Gadget in Kubernetes.

Reply:
True. But I prefer not to hanldle it in the same PR. I wanted to have look into `crio` hook mode and do some testing for IG with this.  I opened the [issue](https://github.com/inspektor-gadget/inspektor-gadget/issues/1142) to track this. What do you think?